data/sql/updates/pending_db_world/rev_1609681055620795500.sql

```diff
@@ -0,0 +1,122 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609681055620795500');
+/*
+ * Raid: The Naxxramas
+ * Update by Knindza | <www.azerothcore.org>
+ * Copyright (C) <www.shadowburn.net> & <www.lichbane.com>
```

Comment:
better remove this as they will be deleted anyway, you can add them in the PR description which instead stays there forever

Reply:
Sorted xP

It's my fix template.